<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_ampsWidgetAutheticateUser</class>
 <widget class="QWidget" name="Form_ampsWidgetAutheticateUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="m_labelUserName">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>60</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_labelPassword">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>100</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_lineEditUsername">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_lineEditPassword">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pushButtonOK">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_labelTitle">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter username &amp; password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
